.github/workflows/android_x64.yml

```diff
@@ -0,0 +1,210 @@
+      SCCACHE_IGNORE_SERVER_IO_ERROR: 1
+      SCCACHE_SERVER_PORT: 35555
+      SCCACHE_ERROR_LOG: /__w/openvino/sccache_log.txt
+      SCCACHE_LOG: warn
+      OPENVINO_REPO: '/__w/openvino/openvino/openvino'
+      VCPKG_ROOT: '/__w/openvino/openvino/vcpkg'
+      BUILD_DIR: '/__w/openvino/openvino/build'
+      ANDROID_TOOLS: '/deps/android_tools'
+      ANDROID_NDK_HOME: '/deps/android_tools/ndk-bundle'
+      ANDROID_SDK_VERSION: 29
+      ANDROID_ABI_CONFIG: x86_64
+      VCPKG_TARGET_TRIPLET: x64-android
+      VCPKG_DEFAULT_BINARY_CACHE: '/mount/caches/ccache/android_x64/vcpkg_cache'
+      VCPKG_FORCE_SYSTEM_BINARIES: '1'
+      SCCACHE_AZURE_KEY_PREFIX: android_x64
+    if: "!needs.smart_ci.outputs.skip_workflow"
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          path: 'openvino'
+
+      - name: Get VCPKG version and put it into GitHub ENV
+        run: echo VCPKG_VERSION=$(cat ${OPENVINO_REPO}/vcpkg.json | python3 -c "import sys, json; print(json.load(sys.stdin)['builtin-baseline'])") >> $GITHUB_ENV
+
+      - name: Init submodules for non vcpkg dependencies
+        run: |
+          pushd ${OPENVINO_REPO}
+            git submodule update --init -- ${OPENVINO_REPO}/src/plugins
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/zlib
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/json
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/gtest
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/gflags
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/xbyak
+            git submodule update --init --recursive -- ${OPENVINO_REPO}/thirdparty/ocl
+          popd
+
+      - name: Clone vcpkg
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          repository: 'microsoft/vcpkg'
+          ref: ${{ env.VCPKG_VERSION }}
+          path: 'vcpkg'
+          fetch-depth: '0'
+
+      #
+      # Print system info
+      #
+
+      - name: System info
+        uses: ./openvino/.github/actions/system_info
+
+      #
+      # Build
+      #
+
+      - name: Build vcpkg
+        run: |
+          mkdir -p ${VCPKG_DEFAULT_BINARY_CACHE}
+          ${VCPKG_ROOT}/bootstrap-vcpkg.sh
+          # patch vcpkg default toolchain to build only Release configuration
+          echo "set(VCPKG_BUILD_TYPE release)" >> ${VCPKG_ROOT}/triplets/x64-android.cmake
+
+      - name: CMake - configure
+        run: |
+          cmake \
+            -G '${{ env.CMAKE_GENERATOR }}' \
+            -DENABLE_INTEL_GPU=ON \
+            -DENABLE_TESTS=ON \
+            -DENABLE_SYSTEM_OPENCL=ON \
+            -DENABLE_SYSTEM_PROTOBUF=ON \
+            -DENABLE_SYSTEM_PUGIXML=ON \
+            -DENABLE_SYSTEM_SNAPPY=ON \
+            -DENABLE_SYSTEM_TBB=ON \
+            -DENABLE_INTEL_NPU=OFF \
```

Comment:
let's add a condition to cmake to enable NPU only Linux and Windows

Reply:
Done, please take a look.